tools/flang2/flang2exe/lldebug.cpp

```diff
@@ -1810,7 +1810,9 @@ lldbg_emit_lexical_block(LL_DebugInfo *db, int sptr, int lineno, int findex,
   NEEDB((db->blk_idx + 1), db->blk_tab, BLKINFO, db->blk_tab_size,
         (db->blk_tab_size + 64));
   db->blk_tab[db->blk_idx].mdnode =
-      lldbg_create_block_mdnode(db, parent_blk_mdnode, lineno, 1, findex, ID++);
+      STYPEG(sptr) == ST_BLOCK ? lldbg_create_block_mdnode(db,
+		      parent_blk_mdnode, lineno, 1, findex, ID++)
```

Comment:
There are tabs here I think. Could you check and remove?

Reply:
Not visible from diff, but after setting list there is some special character "^I". Thanks for mentioning this!
Anyway should I squash(all the commits) it or do an extra commit for this formatting correction ?

Reply:
Please go ahead and squash.